### ISSUE TITLE
downsample_scales？

### ISSUE THREAD

**Issue by a897456** [facebookresearch/AudioDec#26]
https://github.com/facebookresearch/AudioDec/blob/5ec3ab9d53cff2f4d92163c0624a277d173703a5/config/autoencoder/symAD_libritts_24000_hop300.yaml#L61

https://github.com/facebookresearch/AudioDec/blob/5ec3ab9d53cff2f4d92163c0624a277d173703a5/config/autoencoder/symAD_libritts_24000_hop300.yaml#L72

Sorry, what does `downsample_scales` mean?
I initially thought it corresponded to the number of cycles of MPD, but MSD only has 3 `scales`, but `downsample_scales` is also 5.
So I don't understand what this parameter does?

**Comment by a897456** [comment]
> Sorry, what does `downsample_scales` mean? I initially thought it corresponded to the number of cycles of MPD, but MSD only has 3 `scales`, but `downsample_scales` is also 5. So I don't understand what this parameter does?

Sorry, I get it. It is the downsampling parameter for each period or each scale.

Next question
1. Do you know why the period selection is [2 3 5 7 11]? Can it be replaced with other numbers? For example, increasing the number, decreasing the number, or changing numbers.
2. As far as I know, according to the short-term stability of speech, the fundamental frequency of harmonics and the number of harmonics, it can reach 23, [2 3 5 7 11 13 17 19 23]. I don’t know if my understanding is correct.

**Comment by a897456** [comment]
> 2. I don't see any strong correlations between MPD and pitch. It is more like an engineering trick. You can use as many different periods as possible, but it will significantly increase the training cost.

I tried 4 sets of cycles [2 3 5 7], which performed the same as the original HIFIGAN. I also tried 6 sets of cycles [2 3 5 7 11 13], and its performance was not as good as the original HIFIGAN.
I always thought that HIFIGAN's MPD was generated because of the harmonic model, but the 5 groups of periods [2 3 5 7 11] seem to have nothing to do with the harmonic model.
